### DISCUSSION TITLE
Inverse Fullwidth module on desktop

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#885] [Proposals]
CPS has pointed out a doubt about the Movistar+ module of the hub in desktop. Right now the full width section background  is not supported and devs say this behaviour should be defined by design core.

<img width="711" alt="image" src="https://user-images.githubusercontent.com/95479574/199479734-41e7b884-d671-4f70-8bd4-4bb8a8f06183.png">

Teams discussion:
[12:11] Carlos Fernandez Piñeiro
    en movil hay un pequeño hack de "NegativeBox" [Ojo, yo le llamo hack, pero es la forma de trabajar propuesta por mistica] q pone margenes negativos e invierte el responsive layout
Edited​[12:11] Carlos Fernandez Piñeiro
    pero claro, ahi es pq se sabe q en movil el layout le ha aplicado 16 px
​[12:11] Carlos Fernandez Piñeiro
    en desktop creo q no hay nada analogo
​[12:12] Carlos Fernandez Piñeiro
    deberia ser algo resuelto por parte de design core, idealmente

**Comment by yceballost** [comment]
Absolutely, we need to include a component to provide a fullwidth background!

**Reply by yceballost** [reply]
Talking with devs, this need is already possible with Mística
https://ibit.ly/tkMf

**Reply by yceballost** [reply]
mistica-web va a pensar cuál es la mejor forma de incorporar esto a Mística. @atabel comentaba que podría incorporarse al `<ResponsiveLayout>` una prop isInverse.

**Reply by MontseTelefonica** [reply]
Para el nuevo concepto de secciones, cuando se seleccione un color oscuro de fondo, todos los módulos incluidos deberían forzarse a ser inverse. Esta prop isInverse podría servir para hacer eso?
Adjunto pantallazo de ejemplo

<img width="802" alt="image" src="https://user-images.githubusercontent.com/95479574/201945801-2e237548-a961-4eaa-a989-b6c820eaaf48.png">

**Reply by yceballost** [reply]
sip, el isInverse es lo que se usa por ejemplo en una navigation bar para pasarla de blanca a color de marca
https://ibit.ly/_flv

o un header
https://ibit.ly/Pi25

**Reply by MontseTelefonica** [reply]
Perfect! claro clarinete

**Reply by aweell** [reply]
https://github.com/Telefonica/mistica-web/pull/600

Ya se permite introducir un `background-color` en el componente responsive layout.

Archivo discussion
